scripts/grt/export.py

```diff
@@ -146,7 +146,7 @@ def annotate(label, human_label=None):
     blacklisted_tools = config['sanitization']['tools']
 
     annotate('export_jobs_start', 'Exporting Jobs')
-    handle_job = io.open(REPORT_BASE + '.jobs.tsv', 'w', encoding='utf-8')
+    handle_job = codecs.open(REPORT_BASE + '.jobs.tsv', 'w', encoding='utf-8')
```

Comment:
Not sure I understand the reason for this change, we are writing only unicodified content to this file handle.

Reply:
On python 3, on python 2 with bytestring literals https://github.com/mvdbeek/galaxy/blob/05b253124ac32eca1521d11a40ad2bc6ff865755/scripts/grt/export.py#L150 and https://github.com/mvdbeek/galaxy/blob/05b253124ac32eca1521d11a40ad2bc6ff865755/scripts/grt/export.py#L184 cause problems

Reply:
OMG, how this ever worked? Can I change this back to use `io` and change those lines to write Unicode?

Reply:
Of course!